### PR TITLE
update docs for data upload

### PR DESCRIPTION
**Pull request by bcolloran**
### Pull Request checklist

- [ ] **Documentation**: If this feature has or requires documentation, the relevant docs have been updated.
- [ ] **Changelog**: This PR updates the [changelog](../CHANGELOG.md) with any user-visible changes.
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not

### CODE REVIEW

File: docs/workflows.md [hunk 1]
```diff
@@ -15,6 +19,25 @@ Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remot
...
 
+### Uploading data to an Iodide notebook
+
+_Note (2019-03-11): we are aware that this workflow is currently quite awkward. It is under active development and will be substantially improved within the next few weeks._
+
+_Note (2019-03-11): during our alpha period, files larger than 10MB cannot be uploaded._
+
+If you want to work with data stored on your local machine, or if you are encountering CORS problems while attempting to use a dataset from a remote serve, you may upload data directly to Iodide.
+
+Data is uploaded on a per-notebook basis -- rather than uploading data to your user account, you upload it to a specific notebook. To do so, go to the "revisions" page of you notebook, which is available at `iodide.io/{path to your notebook}/revisions`. On this page, click the gear icon and select "Upload a File". Once you've uploaded the file, it will be listed under the "Files" heading on this page.
```
Comment (hamilton) [inline docs/workflows.md]: page of YOUR notebook

Comment (bcolloran) [reply docs/workflows.md]: TY!

File: docs/workflows.md [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
 
-### CORS problems
+### From a remote server
+
+It's possible to download data from a remote server using a browser APIs or a fetch cell. Examples are given in the fetch chunk section of the [JSMD docs](jsmd.md).
+
+#### CORS problems
 
 Note that the one of biggest problems people encounter when attempting to load data into Iodide is problems with [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), which can restrict the ability of Iodide to download data from some other websites. Unfortunately, for security reasons browsers make it [impossible to surface CORS errors in a web app](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Functional_overview), so Iodide cannot give you the guidance you need directly, but if you have trouble loading a data file from a third party server, this is very likely to be the reason why. To check, you can open your browser's developer tools (press `ctrl+shift+i`); in case of a CORS issue, you will see a message like:
```
Comment (wlach) [inline docs/workflows.md]: s/load data into Iodide/load data into Iodide from an external server/ ?

File: docs/workflows.md [hunk 3]
```diff
@@ -15,6 +19,25 @@ Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remot
 
 To work around CORS errors, you need to upload your data to a server that does not impose restrictions on resource sharing. We recommend using [GitHub Gists](https://gist.github.com/), which allows users to upload files that can be downloaded within Iodide. After uploading your data into a gist, you can access the data by copying the URL for the "raw" version of your gist, which can be accessed at `https://gist.githubusercontent.com/{your user name}/{your gist id}/raw`
```
Comment (wlach) [inline docs/workflows.md]: s/We recommend using [GitHub.../We recommend using either the Iodide server for storing data (see below) or [GitHub.../

File: docs/workflows.md [hunk 4]
```diff
@@ -15,6 +19,25 @@ Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remot
...
 
+### Uploading data to an Iodide notebook
+
+_Note (2019-03-11): we are aware that this workflow is currently quite awkward. It is under active development and will be substantially improved within the next few weeks._
```
Comment (wlach) [inline docs/workflows.md]: s/within the next few weeks./soon./ ? Let's not commit to anything.

**Pull request merged**